### ISSUE TITLE
Matlab version, can't issue merge request

### ISSUE THREAD

**Issue by RenoFilla** [heremaps/flexible-polyline#47]
Hi, I have written a Matlab version but am unable to share it. The web interface on github.com denies me with the message that signoff is required, apparently not possible to simply write thst in the commit/request message. Github desktop denies me to even login, probably a 2FA issue of github. Git GUI for Windows allows me all above but denies me with a 403 error, perhaps because I created a new subfolder 'Matlab' which does not yet exist in the master branch.

At this stage I am exhausted and can/will pursue no further without help. If you want it, the file is here: https://se.mathworks.com/matlabcentral/fileexchange/101283-decoding-flexible-polyline-from-here-api

Regard, Reno

**Comment by VeaaC** [comment]
If opening a merge request is a bit too troublesome, you could also try to create a commit with the proposed changes locally on your machine, sign off there and export/upload the patch using Git's email workflow:

```
:~/example$ git clone https://github.com/heremaps/flexible-polyline.git
Cloning into 'flexible-polyline'...
...
Resolving deltas: 100% (156/156), done.
:~/example$ cd flexible-polyline/
:~/example/flexible-polyline$ mkdir matlab
:~/example/flexible-polyline$ touch matlab/README.md
:~/example/flexible-polyline$ # add more files
:~/example/flexible-polyline$ git add matlab/
:~/example/flexible-polyline$ git commit -m "Add a Matlab implementation" --signoff
[master 0a42e65] Add a Matlab implementation
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 matlab/README.md
:~/example/flexible-polyline$ # add more commits

:~/example/flexible-polyline$ git format-patch origin
0001-Add-a-Matlab-implementation.patch

# example content
:~/example/flexible-polyline$ cat *.patch
From 0a42e65f5ea59f28736c3b3bf1d1474f6961813d Mon Sep 17 00:00:00 2001
From: Christian Vetter <christian.vetter@here.com>
Date: Wed, 3 Nov 2021 11:53:28 +0100
Subject: [PATCH] Add a Matlab implementation

Signed-off-by: Christian Vetter <christian.vetter@here.com>
---
 matlab/README.md | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 matlab/README.md

diff --git a/matlab/README.md b/matlab/README.md
new file mode 100644
index 0000000..e69de29
-- 
2.30.2
 
 ```

**Comment by VeaaC** [comment]
That being said; If you are just lacking the signoff for https://github.com/heremaps/flexible-polyline/pull/46, then you can usually remedy this with something like:

```
git rebase HEAD~5 --signoff
git push --force
```

**Comment by lene** [comment]
Hi Reno, first, thank you for our contribution! There seem to be several issues here, and I am somewhat confused by your message. First, the signoff problem can be solved like @VeaaC  said, it is also explained in more detail at https://github.com/heremaps/flexible-polyline/pull/46/checks?check_run_id=4046699393 - but a missing signoff should not prevent you from pushing a commit, just from merging the PR. (You cannot merge either way, only project maintainers can do that.)
I cannot comment on your authentication issues with GitHub. I neither use the GitHub Desktop interface nor the Windows GUI client. Have you tried using `git push` from the command line?

**Comment by RenoFilla** [comment]
Hi all, thanks for your quick and helpful replies. I have tried to start anew with pull request #48, which also failed but then I did all git commands as recommended above and in the fail message. At least now it got a green check mark: https://github.com/heremaps/flexible-polyline/pull/48#partial-pull-merging
I hope this is all OK now. I am a newbie to git, if this wasn't already bleeding obvious... 😂
